client/src/containers/profile/index.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import { bindActionCreators } from 'redux'
+import { connect } from 'react-redux'
+import { Divider, 
+          Avatar,
+          RaisedButton 
+        } from 'material-ui'
+import { push } from 'react-router-redux'
+
+import './style.css'
+
+const Profile = props => {
+	const {
+		user: {
+			firstName,
+			lastName,
+			experience,
+      linkedIn,
+      email
+		}, changePage
+	} = props
+
+  return (
+		<div>
+			<h1 className='title'>{firstName} {lastName}</h1>
+      <div className='title'><Avatar size={60} src="./blank-portrait.png" /></div>
+      <div className='experience'>{experience[0][0]} at {experience[0][1]} from {experience[0][2]} to {experience[0][3]}</div>
```

Comment:
is this assuming that the user has a fixed number of experiences? Also, I think the only information that we want to include in the profile view is their personal information and their current position. Basically, the data fields I have in the personal edit page, that way the experience information is only a part of the roadmap.

Reply:
Nah, the number of experiences was just to show some number of past experiences. If we aren't going to show past experiences on the profile view page, how else are we going to textually show past experiences? People can look at the roadmap, but thats graphical as opposed to textual.